### PR TITLE
150 update implemented metrics readme

### PR DESCRIPTION
**Pull request by victornoel**
This is for #150 

I tracked the references in the tickets where the metrics were introduced, and since some of them had no available pdf on the web, I decided to directly link the pdf in this repository instead.

So I applied the same to all other metrics as well as fixed markdown and other errors.

The README can be seen rendered here: https://github.com/victornoel/jpeek/tree/150-update-implemented-metrics-readme

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#435](https://github.com/yegor256/jpeek/issues/435) is assigned to @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/008300ec-928c-4aac-955a-81952d95439a); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by fanifieiev** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/jpeek/issues/435#issuecomment-606169029))

@fanifieiev The user @fanifieiev[/z](https://www.0crat.com/u/fanifieiev) resigned from [#435](https://github.com/yegor256/jpeek/issues/435), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/jpeek/issues/435#issuecomment-606169029))

@fanifieiev Job refused in 0 hours - no penalty, see [§6](http://www.zerocracy.com/policy.html#6)

**Comment by 0crat** [comment]
This pull request [#435](https://github.com/yegor256/jpeek/issues/435) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/25549f7b-c793-480f-8610-35888f58cc7f); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by Vatavuk** [comment]
@paulodamaso should we close #433 ?

**Comment by Vatavuk** [comment]
@paulodamaso ping

**Comment by paulodamaso** [comment]
@Vatavuk Please go on with this review, I've closed #433

File: README.md [hunk 1]
```diff
@@ -110,84 +110,105 @@ _Revisiting Class Cohesion: An empirical investigation on several systems_,<br/>
 Journal of Object Technology, vol.7, no.6, 2008,
 [PDF](http://www.jot.fm/issues/issue_2008_07/article1.pdf).
 
-Here is a list of metrics we already implement:
+### Here is a list of metrics we already implement:
```
Comment (Vatavuk) [inline README.md:113]: @victornoel I believe it should be `we've already implemented`

Comment (victornoel) [reply README.md:113]: @vatavuk both are correct english, but I will change, it is same the for me :)

**Comment by victornoel** [comment]
@Vatavuk thanks, it is done

**Comment by paulodamaso** [comment]
@victornoel build is failing (?) please take a look

**Comment by victornoel** [comment]
@paulodamaso clearly this is not related to the content of this PR, there seem to be an error with the maven build… maybe it is a transient error? Could you try re-trigger the build?

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will find at least 3 issues during next CR; also, keep in mind that these have to be design problems, but not rather cosmetic issues like localisation

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by victornoel** [comment]
@sereshqua actually it worked, it is as @g4s8 said in https://github.com/zerocracy/farm/issues/2252, it should be fixed the next time you get a QA job :)

**Comment by sereshqua** [comment]
@victornoel do you mean QA label or?

should NOT we see smth like this already? 

Screenshot 2020-04-03 at 12 34 19

**Comment by 0crat** [comment]
> @0crat quality acceptable  ([here](https://github.com/yegor256/jpeek/issues/435#issuecomment-608333847))

@sereshqua There is no quality review for [#435](https://github.com/yegor256/jpeek/issues/435), no performer

**Comment by victornoel** [comment]
> @victornoel do you mean QA label or?
> 
> should NOT we see smth like this already?
> 
> Screenshot 2020-04-03 at 12 34 19

@sereshqua well, not here, because the QA here happened before the fix was merged

**Comment by sereshqua** [comment]
@victornoel look at above, I think now it is working properly :D

**Comment by victornoel** [comment]
@sereshqua ah yes, very cool :)

**Pull request merged**